includes/Shopping/Product.php

```diff
@@ -26,10 +26,12 @@
 
 namespace Google\Web_Stories\Shopping;
 
+use JsonSerializable;
+
 /**
  * Class Product
  */
-class Product {
+class Product implements JsonSerializable {
```

Comment:
What is this used for and why is it needed?

Reply:
See PR description. This is what I mentioned in our sync yesterday.

I wanted a way to verify that the Shopify provider returns valid products, so I came up with this idea to serialize each `Product` to JSON and then validate that against the schema.